CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+﻿# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+Prefix your items with `(Template)` if the change is about the template and not the resulting application.
+
+## Before Initial Release
+
+### Added
+- Basic project file structure.
+- Basic empty Flutter project.
+- Basic PowerShell script to create project based on this template.
+- Basic documentation based on the `UnoApplicationTemplate`.
+- Target iOS 17 with a minimum version of iOS 15.
+- Target Android 14 with a minimum version of Android 11.
+
+### Changed
```

Comment:
I don't think we need those sub-sections.
They're not really used in our other template and they bring more confusion than anything else.